packages/calypso-products/src/features-list.tsx

```diff
@@ -2767,4 +2784,88 @@ export const FEATURES_LIST: FeatureList = {
+		getTitle: () => i18n.translate( '20 premium themes' ),
+	},
+	[ FEATURE_48_PREMIUM_THEMES ]: {
+		getSlug: () => FEATURE_48_PREMIUM_THEMES,
+		getTitle: () => i18n.translate( '48 premium themes' ),
+	},
+	[ FEATURE_AD_SUPPORTED_EXPERIENCE ]: {
+		getSlug: () => FEATURE_AD_SUPPORTED_EXPERIENCE,
+		getTitle: () => i18n.translate( 'Ad-supported experience' ),
+	},
+	[ FEATURE_REAL_TIME_STATS ]: {
+		getSlug: () => FEATURE_REAL_TIME_STATS,
+		getTitle: () => i18n.translate( 'Advanced Real Time stats' ),
+	},
+	[ FEATURE_AUTOMATED_BURST_SCALING ]: {
+		getSlug: () => FEATURE_AUTOMATED_BURST_SCALING,
+		getTitle: () => i18n.translate( 'Automated burst scaling' ),
+	},
+	[ FEATURE_DATABASE_ACCESS ]: {
+		getSlug: () => FEATURE_DATABASE_ACCESS,
+		getTitle: () => i18n.translate( 'Database Access' ),
+	},
+	[ FEATURE_DEVELOPER_TOOLS ]: {
+		getSlug: () => FEATURE_DEVELOPER_TOOLS,
+		getTitle: () => i18n.translate( 'Developer tools' ),
+	},
+	[ FEATURE_DYNAMIC_PRODUCT_UPSELLS ]: {
+		getSlug: () => FEATURE_DYNAMIC_PRODUCT_UPSELLS,
+		getTitle: () => i18n.translate( 'Dynamic product upsells' ),
+	},
+	[ FEATURE_FREE_MIGRATIONS ]: {
+		getSlug: () => FEATURE_FREE_MIGRATIONS,
+		getTitle: () => i18n.translate( 'Free migrations' ),
+	},
+	[ FEATURE_FULL_DATA_CENTER_REDUNDANCIES ]: {
+		getSlug: () => FEATURE_FULL_DATA_CENTER_REDUNDANCIES,
+		getTitle: () => i18n.translate( 'Full data center redundancies' ),
+	},
+	[ FEATURE_GITHUB_DEPLOYMENTS ]: {
+		getSlug: () => FEATURE_GITHUB_DEPLOYMENTS,
+		getTitle: () => i18n.translate( 'GitHub deployments' ),
+	},
+	[ FEATURE_HELP_CENTER_SUPPORT ]: {
+		getSlug: () => FEATURE_HELP_CENTER_SUPPORT,
+		getTitle: () => i18n.translate( 'Help center support' ),
+	},
+	[ FEATURE_LIMITED_STATS ]: {
+		getSlug: () => FEATURE_LIMITED_STATS,
+		getTitle: () => i18n.translate( 'Limited stats' ),
+	},
+	[ FEATURE_LOCAL_DEVELOPMENT_ENVIRONMENT ]: {
+		getSlug: () => FEATURE_LOCAL_DEVELOPMENT_ENVIRONMENT,
+		getTitle: () => i18n.translate( 'Local development environment' ),
+	},
+	[ FEATURE_PRE_INSTALLED_SECURITY_PERF_PLUGINS ]: {
+		getSlug: () => FEATURE_PRE_INSTALLED_SECURITY_PERF_PLUGINS,
+		getTitle: () => i18n.translate( 'Pre-installed plugins for security and performance' ),
+	},
+	[ FEATURE_WEB_SERVER_SETTINGS ]: {
+		getSlug: () => FEATURE_WEB_SERVER_SETTINGS,
+		getTitle: () => i18n.translate( 'Web server settings' ),
+	},
+	/* END: Trail Map New Features */
 };
+
+FEATURES_LIST[ FEATURE_PRE_INSTALLED_ECOMMERCE_PLUGINS ] = {
+	...FEATURES_LIST[ FEATURE_PRE_INSTALLED_ECOMMERCE_PLUGINS ],
+	getSubFeatures: () => ( {
+		[ FEATURE_UNLIMITED_PRODUCTS ]: FEATURES_LIST[ FEATURE_UNLIMITED_PRODUCTS ],
+		[ FEATURE_MIN_MAX_ORDER_QUANTITY ]: FEATURES_LIST[ FEATURE_MIN_MAX_ORDER_QUANTITY ],
+		[ FEATURE_BULK_DISCOUNTS ]: FEATURES_LIST[ FEATURE_BULK_DISCOUNTS ],
+		[ FEATURE_INVENTORY ]: FEATURES_LIST[ FEATURE_INVENTORY ],
+		[ FEATURE_DYNAMIC_PRODUCT_UPSELLS ]: FEATURES_LIST[ FEATURE_DYNAMIC_PRODUCT_UPSELLS ],
+	} ),
+};
```

Comment:
p.s. different topic - you can start using https://github.com/Automattic/wp-calypso/pull/89650 for testing the categorised layout. But no need to rebase any other PR over it.

Reply:
Well ideally I'd expect just slugs, but if it will make it easier to connect things, then let's keep it and we can follow up on it later. :-)

Reply:
I guess just my only curiosity is whether these are actual "sub features" or just some extended description. Like if we end up adding more generalised things in there, then it may start to loose that semantic/context.